tools/merge_requirements.py

```diff
@@ -16,17 +17,22 @@ def process_entries(entries):
 
     :param list entries: List of entries
 
-    :returns: mapping from a project to its pinned version
+    :returns: mapping from a project to its version specifier
     :rtype: dict
     """
     data = {}
     for e in entries:
         e = e.strip()
         if e and not e.startswith('#') and not e.startswith('-e'):
-            project, version = e.split('==')
-            if not version:
+            for comparison in ('==', '<=',):
```

Comment:
Can we add a TODO to remove this code once it is not needed anymore. I do not think it is worth to keep this capability to use `<=` when the ultimate goal is to consume pinned dependencies here.

Reply:
Done